react/components/EditorContainer/Sidebar/ConfigurationList/ContentEditor/ConditionControls/Scheduler.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { Component } from 'react'
+import { InjectedIntlProps, injectIntl } from 'react-intl'
+import { withRuntimeContext } from 'vtex.render-runtime'
+import { DatePicker, Toggle } from 'vtex.styleguide'
+
+import {
+  getFirstTime,
+  getLastTime,
+  getNextDay,
+  getNextTime,
+  getPreviousTime,
+  isLastTime,
+  isSameDay,
+} from '../../../../../../utils/date'
+
+import ConditionTitle from './ConditionTitle'
+import Typings from './typings'
```

Comment:
Any reason to import this as `Typings` and use `Typings.DateRange` instead of importing directly as `{ DateRange }`? If none, consider changing this `import` statement 🍡

Reply:
As we're using a lot of global types, I thought it'd be better to identify the local ones somehow; I've also used that "convention" in a few other files.

Reply:
Ok! I found it strange because it was only one type definition, but if that's the motivation, I think it's fair